.gitignore

```diff
@@ -108,6 +108,8 @@ venv/
 ENV/
 env.bak/
 venv.bak/
+Pipfile
```

Comment:
What will these two be ignoring?

Reply:
i created the virtual environment using "pipenv" these two are the configuration files for it.